pkg/bundle/keychain.go

```diff
@@ -0,0 +1,77 @@
+package bundle
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sync"
+
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/cli/cli/config/types"
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
do we need to import docker homedir just to get the homedir? we already have github.com/mitchellh/go-homedir maybe this can be used instead?

Reply:
Yes we can use `go-homedir` here.